Dockerfile

```diff
@@ -25,6 +25,9 @@ ENV USER root
 # WARNING: disabling this may slow down a lot your builds!
 ENV USE_CCACHE 1
 
+# We need to specify the ccache binary since it is no longer packaged along with AOSP
+ENV CACHE_EXEC /usr/bin/ccache
```

Comment:
You should replace `CACHE_EXEC` to `CCACHE_EXEC` if you want to see `ccache` working. :)

Reply:
Thanks for catching that! Fixed in ea1282f